src/Tests.Microsoft.Security.Utilities.Core/SecretMaskerTests.cs

```diff
@@ -730,10 +739,24 @@ public void SecretMasker_NotAddShortEncodedSecrets()
         var input = "ab123cd345";
         var result = secretMasker.MaskSecrets(input);
 
-        Assert.AreEqual("ab***cd***", result);
+        Assert.AreEqual("abyyycdyyy", result);
     }
 
 
+    [TestMethod]
+    public void SecretMasker_DistinguishLiteralAndRegexRedactionTokens()
```

Comment:
>SecretMasker_DistinguishLiteralAndRegexRedactionTokens [](http://example.com/codeflow?start=16&length=54)

oh, look, i *did* author a test ensuring that these two classes of redaction token are honored. :)